tests/components/mazda/test_button.py

```diff
@@ -65,40 +65,6 @@ async def test_button_setup_electric_vehicle(hass: HomeAssistant) -> None:
 
     entity_registry = er.async_get(hass)
 
-    entry = entity_registry.async_get("button.my_mazda3_start_engine")
```

Comment:
Would adding tests that check if the entities are not present if the vehicle is electric be benificial?

Reply:
Good idea. I just added tests for that.